setting:
  config_0:
    function: move
    method: cmd_vel
    target: base_foot
    topic: /RTRDoubleArmV7/base_controller/cmd_vel

  config_1:
    function: work
    method: jog_frame
    target: tohoku_gipper
    topic: /jog_frame

  config_2:
    function: work
    method: jog_frame
    target: mani_gipper
    topic: /jog_frame
